lale/operators.py

```diff
@@ -3446,6 +3721,25 @@ def __call__(self, steps: List[Any]):
 Pipeline = _PipelineFactory()
 
 
+def pipeline_graph_class(steps) -> Type[PlannedPipeline]:
```

Comment:
Maybe add a leading underscore to this function name too.

Reply:
done